tests/conftest.py

```diff
@@ -117,24 +117,12 @@ def sync_http_client_for_test():
     return http_client
 
 
-@pytest.fixture
-def async_http_client_for_test():
-    _, _, http_client = _get_test_client_setup("AsyncHTTPClient")
-    return http_client
```

Comment:
Not used. Sync versions are used for sync-only modules, but could eventually be removed.